packages/container/lib/registry.js

```diff
@@ -861,7 +861,7 @@ function has(registry, fullName, source) {
 }
 
 const privateNames = dictionary(null);
-const privateSuffix = `${Math.random()}${Date.now()}`;
+const privateSuffix = `${Math.random()}${Date.now()}`.replace('.', '');
```

Comment:
Seems fine, but I'm surprised that we don't consider `Date.now()` to be private enough....